packages/react-components/react-charts-preview/src/components/LineChart/LineChart.tsx

```diff
@@ -1298,7 +1299,7 @@ export const LineChart: React.FunctionComponent<ILineChartProps> = React.forward
         xAxisType={isXAxisDateType ? XAxisTypes.DateAxis : XAxisTypes.NumericAxis}
         customizedCallout={_getCustomizedCallout()}
         onChartMouseLeave={_handleChartMouseLeave}
-        enableFirstRenderOptimization={props.enablePerfOptimization && _firstRenderOptimization}
+        //enableFirstRenderOptimization={props.enablePerfOptimization && _firstRenderOptimization}
```

Comment:
instead of commenting this code set default value of _firstRenderOptimization = false

Reply:
Okay